### DISCUSSION TITLE
普通用户没有SQL查询导出按钮

### DISCUSSION THREAD

**Discussion by niduyi** [hhyo/Archery#1771] [Q&A]
因升级，普通用户没有SQL查询的结果导出按钮了，请问，怎么把按钮配置出来？

**Comment by niduyi** [comment]
sql | permission | 在线查询下载权限

为什么我没有看到有这个权限分类？

**Comment by nick2wang** [comment]
升级时记得跑src/init_sql/下的sql文件
https://github.com/hhyo/Archery/blob/6bb36f721481deaff1a29c29c26cd674d02620a3/src/init_sql/v1.8.3.sql#L28-L31

**Comment by niduyi** [comment]
嗯，开始翻看旧的版本找到了这个权限控制，我是直接升级到1.8.5然后又用旧版本的数据覆盖新版本数据的，所以导致数据被覆盖掉， 。
